### ISSUE TITLE
Role radio should have required prop aria-checked

### ISSUE THREAD

**Issue by badtant** [evcohen/eslint-plugin-jsx-a11y#69]
Role checkbox has this required prop and I can't see why radio shouldn't.
According to W3C this is a required prop: https://www.w3.org/TR/wai-aria/roles#radio

**Comment by evcohen** [comment]
good find - thanks!
